templates/articles/focused_article_page.html

```diff
@@ -45,13 +45,35 @@
         </div>
     </div>
 
-    {% if page.primary_topic or page.primary_time_period %}
-        <div class="related-content related-content--borderless">
-            <div class="container">
+    <div class="related-content related-content--borderless">
```

Comment:
Would we always get this `<div>` regardless of whether there is anything in it?

Reply:
I was thinking that @ahosgood, but decided that running the logic to check if 1 of 3 scenarios would be occurring is more than it's worth. There will _always_ be `latest_items` by nature of how it works, and there will usually be a related topic/time period too 😃